components/new-contribution-flow/ContributionFlowSuccess.js

```diff
@@ -232,113 +245,161 @@ class NewContributionFlowSuccess extends React.Component {
+              flexShrink={0}
+            >
+              <Flex flexDirection="column" alignItems="center" justifyContent="center" my={4} width={1}>
+                <H3 mb={3}>
+                  <FormattedMessage id="NewContributionFlow.Success.Header" defaultMessage="Thank you! 🎉" />
+                </H3>
+                <Box mb={3}>
+                  <P fontSize="20px" fontColor="black.700" fontWeight={500} textAlign="center">
+                    <FormattedMessage
+                      id="NewContributionFlow.Success.NowSupporting"
+                      defaultMessage="You are now supporting {collective}."
+                      values={{ collective: order.toAccount.name }}
+                    />
+                  </P>
+                </Box>
+                <ContributorCardWithTier width={250} height={380} contribution={order} my={2} />
+                <Box my={4}>
+                  <P fontColor="black.800" fontWeight={500}>
+                    <FormattedMessage
+                      id="NewContributionFlow.Success.DiscoverMore"
+                      defaultMessage="Discover more Collectives like {collective} &rarr;"
+                      values={{ collective: order.toAccount.name }}
+                    />
+                  </P>
+                </Box>
+                <Flex justifyContent="center" mt={2}>
+                  <ShareLink
+                    href={tweetURL({
+                      url: shareURL,
+                      text: `I've just donated to {order.toAccount.name}`,
+                    })}
+                  >
+                    <Twitter size="1.2em" color="#4E5052" />
+                    <FormattedMessage id="tweetIt" defaultMessage="Tweet it" />
+                  </ShareLink>
+                  <ShareLink href={facebookShareURL({ url: shareURL })}>
+                    <Facebook size="1.2em" color="#4E5052" />
+                    <FormattedMessage id="shareIt" defaultMessage="Share it" />
+                  </ShareLink>
+                </Flex>
+                {this.renderCommentForm(LoggedInUser, getPublicMessage(order))}
+              </Flex>
+            </ContainerWithImage>
+            <Flex flexDirection="column" alignItems="center" justifyContent="center" width={1}>
+              {pendingOrder ? this.renderBankTransferInformation() : this.renderCallsToAction()}
             </Flex>
-            {this.renderCommentForm(LoggedInUser, getPublicMessage(order))}
-          </Flex>
-        </ContainerWithImage>
-        <Flex flexDirection="column" alignItems="center" justifyContent="center" width={[1, 1 / 2]}>
-          {this.renderCallsToAction()}
-        </Flex>
+          </Fragment>
+        )}
       </Flex>
     );
   }
 }
 
 // GraphQL
 const orderSuccessQuery = gqlV2/* GraphQL */ `
-  query NewContributionFlowOrderSuccess($order: OrderReferenceInput!, $memberAccount: AccountReferenceInput) {
+  query NewContributionFlowOrderSuccess($order: OrderReferenceInput!) {
```

Comment:
How do we retrieve the public message if we remove the `memberAccount`?

Reply:
Adding `fromAccountMember` field, see associated API PR

Reply:
https://github.com/opencollective/opencollective-api/pull/4501